akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala

```diff
@@ -32,10 +36,9 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
   @volatile
   private var _reachability: Reachability = Reachability.empty
 
-  // lazy init below, updated when state is updated
   @volatile
-  private var _cachedSelf: OptionVal[Member] = OptionVal.None
-
+  private var _cachedSelf: Member =
+    Member(cluster.selfUniqueAddress, cluster.selfRoles).copy(status = MemberStatus.Removed)
```

Comment:
starting as `Removed`, which was the case previously also, via `selfFromStateOrPlaceholder`

Reply:
`MemberStatus.Removed` indicates history, is cached on first startup = no history?

Reply:
This PR is not changing that behavior. I agree that it can be confusing that it's status is Removed before join. Then we would have to add another initial member status. That should be a separate PR, and we already have issue https://github.com/akka/akka/issues/25663 for it.

Reply:
OK perfect, better that it is separate and we have it set to do.